app/utils/getWarningMessage.js

```diff
@@ -64,6 +68,33 @@ export default flags => {
         </Col>
       </Row>
     );
+  } else if (flags.replaced) {
+    warningMessage = (
+      <Row>
+        <Col sm>
+          <StyledCard inverse>
+            <CardHeader
+              style={{
+                backgroundColor: '#a94442',
+                borderColor: '#a94442',
+              }}
+            >
+              Warning: This property is deprecated.
+            </CardHeader>
+            <StyledCardBody>
+              <CardText>
+                Please note that the issuer of this property - Agoras Token (#35),
```

Comment:
@germanf we need for the text 'Agoras Token #35 ' to be dynamic based on the actual current token Name/ Property ID

Reply:
gotcha!